src/components/projects-summary/projects-summary.controller.ts

```diff
@@ -1,23 +1,144 @@
 import * as angular from 'angular';
+import * as _ from 'lodash';
+import * as jQuery from 'jquery';
```

Comment:
I see we have 'import * as $ from 'jquery';' in the landing-page.controller.js, it would be good to standardize on one or the other.